### DISCUSSION TITLE
Idea Tracker

### DISCUSSION THREAD

**Discussion by WildDogOne** [certeu/droid#12] [Ideas]
- [ ] Rewrite all functions to have named variables (maybe with typecasting)
- [ ] Additional flag for output of search results instead of only hitcount.
- [x] Make logging great again: JSON output needs filepath, also JSON output to stdout for CICD would be great
- [ ] Add Certificate Authentication for Microsoft XDR Backend
- [ ] Add Token Authentication for Elastic Backend
- [ ] Update sentinel and xdr platform to support the latest Kusto backend
- [ ] Add more testcases

## TestCases

- [ ] Converting Sigma into all supported formats
- [ ] Validation of Sigma rules
- [ ] Mock Export
- [ ] Mock Validation?
- [ ] Test logging outputs (Log to File, Log to stdout, Log JSON to stdout)

**Comment by 0xFustang** [comment]
Thanks @WildDogOne, it makes sense! I will add these points to a public Kanban if possible!
